appgate/customloaders.py

```diff
@@ -24,12 +24,38 @@ class CustomLoader:
 class CustomAttribLoader(CustomLoader):
     loader: Callable[[Any], Any] = attrib()
     field: str = attrib()
+    load_external: bool = attrib()
+
+    def load(self, values: AttributesDict) -> AttributesDict:
```

Comment:
Should this be an `async def` method since it is potentially doing I/O?
For blocking I/O you would then use `asyncio.to_thread` to run it in a thread pool.